inc/views/nav_walker.php

```diff
@@ -216,6 +218,61 @@ public function enqueue_mega_menu_style() {
 		wp_style_add_data( 'neve-mega-menu', 'rtl', 'replace' );
 		wp_style_add_data( 'neve-mega-menu', 'suffix', '.min' );
 		wp_enqueue_style( 'neve-mega-menu' );
+
+		$style = <<<'CSS'
+.left .neve-mega-menu > .sub-menu {
+	left: 0;
+	right: auto;
+	transform: none;
+}
+.center .neve-mega-menu > .sub-menu {
+	transform: translateX(50%);
+}
+.right .neve-mega-menu > .sub-menu {
+	left: auto;
+	right: 0;
+	transform: none;
+}
+CSS;
+		$style = Dynamic_Css::minify_css( $style );
+		wp_add_inline_style( 'neve-mega-menu', $style );
+
+		// Fix for MegaMenu alignment
+		$script = <<<'JS'
+function megaMenuCalcEvent() {
+    console.log('Event triggered');
```

Comment:
Can you please remove this console.log over here? 👍🏻

Reply:
I've cleaned the code and removed unnecessary left variables.